server/api/events/all.get.ts

```diff
@@ -1,8 +1,4 @@
-import { PrismaClient } from "@prisma/client";
-
 export default defineEventHandler(async event => {
-  const prisma = new PrismaClient();
-  const records = await prisma.event.findMany();
-  prisma.$disconnect();
+  const records = await event.context.prisma.event.findMany();
```

Comment:
could you not just `return event.context.prisma.event.findMany();` and get rid of the `async`/`await` bits here?

Reply:
absolutely, will do